plugins/csp-vr-accessibility/src/Plugin.cpp

```diff
@@ -270,11 +308,11 @@ void Plugin::onLoad() {
   from_json(mAllSettings->mPlugins.at("csp-vr-accessibility"), *mPluginSettings);
 
   // Create & configure FloorGrid
-  mGrid = std::make_shared<FloorGrid>(mSolarSystem);
-  mGrid->configure(mPluginSettings);
+  mGrid = std::make_shared<FloorGrid>(mSolarSystem, mPluginSettings->mGridSettings);
```

Comment:
Is there a reason, why the `FloorGrid` and `FovVignette` are `shared_ptr` instead of `unique_ptr`?

Reply:
Changing `FloorGrid` or `FoVVignette` to `unique_ptr` causes CosmoScout to crash when either is activated.